src/elements/content-sidebar/activity-feed/common/activity-message/ToggleMoreLessButton.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @flow
+ * @file Show more/less button
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import PlainButton from '../../../../../components/plain-button';
+
+import messages from './messages';
+
+type Props = {
+    isMore: boolean,
+    onClick: Function,
```

Comment:
Can we type this with the arrow syntax `() => void` to be more explicit to what the props and return value are